spec/main_spec.rb

```diff
@@ -5,11 +5,11 @@
     it { should be_installed }
   end
 
-  describe file('/etc/clickhouse-server/config.xml') do
+  describe file('/etc/clickhouse-server/config.d/dirs.xml') do
```

Comment:
@artworklv changed this to dirs.xml although you had changed this to data-paths.xml before (which does not exist anywhere). Hope that was OK.